package-lock.json

```diff
@@ -11043,9 +11043,9 @@
             }
         },
         "node_modules/vscode-azureextensiondev": {
-            "version": "0.9.2",
-            "resolved": "https://registry.npmjs.org/vscode-azureextensiondev/-/vscode-azureextensiondev-0.9.2.tgz",
-            "integrity": "sha512-2TsA5ct2Zzm5ydHv2Vj3Sko6Bm1BRHRc+qOsKDmgL+XQfblY0tyKBSb93HGJVfu4+HKU6MIV7ZUib/RfKOY5Dg==",
+            "version": "0.9.5",
```

Comment:
Why does this say 0.9.5 instead of 0.9.4?

Reply:
That's a very good question 🤔

Reply:
Because the carot in front of "^0.9.4" allows for higher versions as long as it's not a breaking change